### PR TITLE
Added a feature to destroy floor and replicas from server that is di…

### PR DESCRIPTION
**Pull request by CarGuerrero**
…sconnected or lost connection

### CODE REVIEW

File: Source/RN4UE4/RakNetRP.cpp [hunk 1]
```diff
@@ -4,6 +4,7 @@
 #include <functional>
 #include <string>
 #include "ReplicaBase.h"
+#include "Engine.h"
```
Comment (Alexthegr92) [inline Source/RN4UE4/RakNetRP.cpp]: Why is this needed?

File: Source/RN4UE4/RakNetRP.cpp [hunk 2]
```diff
@@ -117,7 +120,7 @@ void ARakNetRP::Tick(float DeltaTime)
 	}
 
 	// TODO: Handle servers disconnecting
-	if (!allServersChecked)
+	if (!allServersChecked && initialised)
```
Comment (Alexthegr92) [inline Source/RN4UE4/RakNetRP.cpp]: Why is initialised needed?

File: Source/RN4UE4/RakNetRP.h [hunk 3]
```diff
@@ -108,6 +110,6 @@ class RN4UE4_API ARakNetRP : public AActor, public ReplicaManager3
 	bool initialised;
 
 	static const int SERVER_PORT = 12345;
-	int						totalServers;
+	int						totalServers = -1;
```
Comment (Alexthegr92) [inline Source/RN4UE4/RakNetRP.h]: Use an initialiser list with the constructor instead of initialising here

File: Source/RN4UE4/Replica.h [hunk 4]
```diff
@@ -12,7 +12,7 @@
 #include "VariableDeltaSerializer.h"
 #include "GetTime.h"
 #include "Rand.h"
-
+#include "Materials/Material.h"
```
Comment (Alexthegr92) [inline Source/RN4UE4/Replica.h]: Why is this needed?

Comment (CarGuerrero) [reply Source/RN4UE4/Replica.h]: It gives an error that UMaterial wasn't found if I don't put the library

File: Source/RN4UE4/RakNetRP.cpp (hunk 1 above)
Comment (CarGuerrero) [reply Source/RN4UE4/RakNetRP.cpp]: It gives an error with GEngine because I didn't recognize it class

Comment (Alexthegr92) [reply Source/RN4UE4/RakNetRP.cpp]: What is new that needs it?

File: Source/RN4UE4/Replica.h (hunk 4 above)
Comment (Alexthegr92) [reply Source/RN4UE4/Replica.h]: What is  new that needs it?

**Review by Alexthegr92 — APPROVED** [review]
